base_suspend_security/models/ir_model_access.py

```diff
@@ -17,19 +17,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from openerp import models, tools
+from odoo import models, api, tools
 from ..base_suspend_security import BaseSuspendSecurityUid
 
 
 class IrModelAccess(models.Model):
     _inherit = 'ir.model.access'
 
-    @tools.ormcache_context('uid', 'model', 'mode', 'raise_exception',
+    @api.model
+    @tools.ormcache_context('self._uid', 'model', 'mode', 'raise_exception',
                             keys=('lang',))
```

Comment:
@hbrunn Why is the 'lang' value into the context a discriminant value to use as cache key?